src/Common/src/Interop/Unix/System.Native/Interop.FileDescriptors.cs

```diff
@@ -1,15 +1,18 @@
 // Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
+using System;
+using Microsoft.Win32.SafeHandles;
+
 internal static partial class Interop
 {
     internal static partial class Sys
     {
         internal static class FileDescriptors
         {
-            internal const int STDIN_FILENO = 0;
-            internal const int STDOUT_FILENO = 1;
-            internal const int STDERR_FILENO = 2;
+            internal static readonly SafeFileHandle STDIN_FILENO = new SafeFileHandle((IntPtr)0, false);
+            internal static readonly SafeFileHandle STDOUT_FILENO = new SafeFileHandle((IntPtr)1, false);
+            internal static readonly SafeFileHandle STDERR_FILENO = new SafeFileHandle((IntPtr)2, false);
```

Comment:
Nit: could you name the Boolean arguments, e.g. ```ownsHandle: false```?  That helps me as a reader to understand very clearly what it means.